src/helpers/commands/delete_slash_response.ts

```diff
@@ -1,17 +1,17 @@
-import { applicationID } from "../../bot.ts";
+import { applicationId } from "../../bot.ts";
 import { RequestManager } from "../../rest/request_manager.ts";
 import { endpoints } from "../../util/constants.ts";
 
 /** To delete your response to a slash command. If a message id is not provided, it will default to deleting the original response. */
-export async function deleteSlashResponse(token: string, messageID?: string) {
+export async function deleteSlashResponse(token: string, messageId?: string) {
   const result = await RequestManager.delete(
-    messageID
-      ? endpoints.INTERACTION_ID_TOKEN_MESSAGEID(
-        applicationID,
+    messageId
+      ? endpoints.INTERACTION_Id_TOKEN_MESSAGEId(
```

Comment:
```suggestion
      ? endpoints.INTERACTION_ID_TOKEN_MESSAGEId(
```

Reply:
Is this correct?

Reply:
I didn't saw the second Id

Reply:
Isn't there supposed be an underscore after `MESSAGE`?
`INTERACTION_ID_TOKEN_MESSAGEID` -> `INTERACTION_ID_TOKEN_MESSAGE_ID`

Reply:
yeah you're right